content/docs/concepts/csidriver/troubleshooting/powerflex.md

```diff
@@ -25,10 +23,8 @@ description: Troubleshooting PowerFlex Driver
-| After installing version v2.6.0 of the driver using the default `powerflexSdc` image, sdc:3.6.0.6, the vxflexos-node pods are in an `Init:CrashLoopBackOff` state. This issue can happen on hosts that require the SDC to be installed manually. Automatic SDC is only supported on Red Hat CoreOS (RHCOS), RHEL 7.9, RHEL 8.4, RHEL 8.6. | The SDC is already installed. Change the `images.powerflexSdc` value to an empty value in the [values](https://github.com/dell/csi-powerflex/blob/72b27acee7553006cc09df97f85405f58478d2e4/helm/csi-vxflexos/values.yaml#L13) and re-install. |
-| After installing version v2.8.0 of the driver using the default `powerflexSdc` image, sdc:3.6.1, the vxflexos-node pods are in an `Init:CrashLoopBackOff` state. This issue can happen on hosts that require the SDC to be installed manually. Automatic SDC is only supported on Red Hat CoreOS (RHCOS), RHEL 7.9, RHEL 8.4, RHEL 8.6. | The SDC is already installed. Change the `images.powerflexSdc` value to an empty value in the [values](https://github.com/dell/csi-powerflex/blob/72b27acee7553006cc09df97f85405f58478d2e4/helm/csi-vxflexos/values.yaml#L13) and re-install. |
-| In version v2.6.0, the driver is crashing because the External Health Monitor sidecar crashes when a persistent volume is not found. | This is a known issue reported at [kubernetes-csi/external-health-monitor#100](https://github.com/kubernetes-csi/external-health-monitor/issues/100). |
-| In version v2.6.0, when a cluster node goes down, the block volumes attached to the node cannot be attached to another node. | This is a known issue reported at [kubernetes-csi/external-attacher#215](https://github.com/kubernetes-csi/external-attacher/issues/215). Workaround: <br/> 1. Force delete the pod running on the node that went down. <br/> 2. Delete the pod's persistent volume attachment on the node that went down. Now the volume can be attached to the new node.
+| After installing version v2.6.0 and v2.8.0 of the driver using the default `powerflexSdc` image, sdc:3.6.0.6 and 3.6.1, the vxflexos-node pods are in an `Init:CrashLoopBackOff` state. This issue can happen on hosts that require the SDC to be installed manually. Automatic SDC is only supported on Red Hat CoreOS (RHCOS), RHEL 7.9, RHEL 8.4, RHEL 8.6. | The SDC is already installed. Change the `images.powerflexSdc` value to an empty value in the [values](https://github.com/dell/csi-powerflex/blob/72b27acee7553006cc09df97f85405f58478d2e4/helm/csi-vxflexos/values.yaml#L13) and re-install. |
+| When a cluster node goes down, the block volumes attached to the node cannot be attached to another node. | This is a known issue reported at [kubernetes-csi/external-attacher#215](https://github.com/kubernetes-csi/external-attacher/issues/215). Workaround: <br/> 1. Force delete the pod running on the node that went down. <br/> 2. Delete the pod's persistent volume attachment on the node that went down. Now the volume can be attached to the new node.
```

Comment:
This related to driver v2.6.0 and v2.8.0. Is it valid for CSM 1.14 as well ? If it's not applicable, we can consider removing it.